server/application/user/UserService.ts

```
@@ -0,0 +1,31 @@
+    const user = await this.userRepository.findById(userId);
+    if (!user) {
+      throw new Error(`User with id ${userId} not found.`);
+    }
+    return user.isPortalConnected();
+  }
+
+  async initializePortalConnection(studentData: StudentInitializationDataType): Promise<void> {
+    const userId = await this.authService.getAuthenticatedUserId();
+    const user = await this.userRepository.findById(userId);
+    if (user && user.isPortalConnected()) {
+      throw new Error('이미 연동된 유저입니다.');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)